app/screens/ArtistView.tsx

```diff
@@ -132,6 +152,12 @@ const ArtistView = React.memo<{ id: string; title: string }>(({ id, title }) =>
 
   const { artist, albums } = artistData
 
+  // If playing all songs, set queue once loading complete
```

Comment:
Honestly, using a local state to control if the queries are enabled and then set the queue feels very roundabout to me.

But I don't know React well enough to know if there's a better solution, Something based around a Promise would seem more logical to me.